layouts/shortcodes/cloudchoose.html

```diff
@@ -1,8 +1,8 @@
 <div class="mdl-tabs cloud-tab-tabs">
     <div class="mdl-tabs__tab-dir cloud-tab-tabstrip">
-        <a class="mdl-tabs__tab cloud-tab is-active">AWS</a>
-        <a class="mdl-tabs__tab cloud-tab">Azure</a>
-        <a class="mdl-tabs__tab cloud-tab">GCP</a>
+        <a data-track="cloudchoose-aws" class="mdl-tabs__tab cloud-tab is-active">AWS</a>
```

Comment:
These raise the question of how we'll use these on the back end, which I think this PR should address somehow as well, since there are so many additions here. For example, if we wanted to ask ourselves the question, "Which cloud do docs users choose most often?", I'd imagine the ideal (pseudo-)query we'd want to write would be something like, "Select all clicks of cloudchoose type, grouped by provider". Would this naming scheme support that kind of query?

Reply:
In a way yes. We can filter out IDs in GA to start so we would only see clicks with "cloudchoose" in the ID and then we can look at the list of events. If we only look at a single page at a time, the above query becomes a bit easier. So yes we would be able to, but it will take a bit of leg work outside of GA in its current state. I think this is a good place to start, and we might have to adjust based on the specific questions we want to have answered.